django/applications/catmaid/control/tile.py

```diff
@@ -109,10 +109,10 @@ def put_tile(request, project_id=None, stack_id=None):
     return HttpResponse("Image pushed to HDF5.", content_type="plain/text")
 
 
-class TileSource(object):
+class TileSource(object): # Is this meant to derive from DefaultTileSource? Otherwise, there's no self.get_tile_url
```

Comment:
will remove this comment before commit

Reply:
The other way around: the other tile sources (e.g. `DefaultTileSource` inherit from `TileSource` and implement this function. This would be clearer though of `TileSource` had the abstract `get_tile_url()` method, possibly raising an error when called.